cosmo_tester/test_suites/snapshots/__init__.py

```diff
@@ -55,6 +56,33 @@
     TENANT_DEPLOYMENTS_PATH + '/{name}'
 )
 
+MANAGER_VERSIONS_LIST = (
+    '3.4.2',
+    '4.0',
+    '4.0.1',
+    '4.1',
+    'master'
+)
+# Only test 3.4.2 and 4.0 with a single tenant on premium.
+# Everything else should work with multi tenant
+MANAGER_SINGLE_TENANT_SLICE = 2
```

Comment:
Hmm. I think the only change that might happen would be to reduce it by 1 (to make it only be 3.4.2), if we were to patch in support for restoring multi tenant snapshots for 4... but I don't think that's likely to happen.

So I think the only changes absolutely needed in future will be to add in new old manager versions (e.g. adding 4.2 before 'master' when we start working on 4.2.1 or 4.3).

I could tweak it to have SINGLE_TENANT and MULTI_TENANT lists, yes, and probably automatically add 'master' to avoid that needing to be messed with in future. I'm not sure there's any way I can fully discourage someone from copying it as an official list of managers (except that it doesn't include the most recent)... will think about it and tweak this bit on Monday AM (hopefully that'll give time for other comments too).